solo-chains/runtime/dancelight/src/lib.rs

```diff
@@ -1634,10 +1637,97 @@ impl pallet_inflation_rewards::Config for Runtime {
     type InflationRate = InflationRate;
     type OnUnbalanced = OnUnbalancedInflation;
     type PendingRewardsAccount = PendingRewardsAccount;
-    type StakingRewardsDistributor = InvulnerableRewardDistribution<Self, Balances, ()>;
+    type StakingRewardsDistributor = InvulnerableRewardDistribution<Self, Balances, PooledStaking>;
     type RewardsPortion = RewardsPortion;
 }
 
+parameter_types! {
+    pub StakingAccount: AccountId32 = PalletId(*b"POOLSTAK").into_account_truncating();
+    pub const InitialManualClaimShareValue: u128 = MILLIUNITS;
+    pub const InitialAutoCompoundingShareValue: u128 = MILLIUNITS;
+    pub const MinimumSelfDelegation: u128 = 10_000 * UNITS;
+    pub const RewardsCollatorCommission: Perbill = Perbill::from_percent(20);
+    // Need to wait 2 sessions before being able to join or leave staking pools
+    pub const StakingSessionDelay: u32 = 2;
+}
+
+pub struct SessionTimer<G>(PhantomData<G>);
```

Comment:
Can you put something more descriptive than `G`? Like `SessionDelay` or `Delay`?

Reply:
*fixed*